packages/connect-iframe/src/index.ts

```diff
@@ -309,6 +309,8 @@ const init = async (payload: IFrameInit['payload'], origin: string) => {
             _popupMessagePort.onmessage = message => handleMessage(message);
         } catch (error) {
             // tell the popup to use MessageChannel fallback communication (thru IFRAME.LOADED > POPUP.INIT)
+            // TODO!!!
+            console.warn('BroadcastChannel error', error);
```

Comment:
do we have analytics at this point? at least sometimes yes I guess. we could use it to report errors, at least numbers of failed inits. and maybe we should finally put sentry here.

Reply:
This actually doesn't seem to be a point of failure. At first I thougt it's unhandled based on the comments, but connect-popup fallbacks to MessageChannel automatically